packages/mainmenu/src/run.ts

```diff
@@ -74,6 +74,8 @@ export namespace IRunMenu {
      */
     runAll?: (widget: T) => Promise<void>;
 
+    runAllMarkdown?: (widget: T) => Promise<void>;
```

Comment:
The `ICodeRunner` interface is meant to expose a way for many different types of things that run code to hook into the "Run" menu. In this case, I think this command is pretty specific to notebooks, and not a generic "code-running" activity. So I'd recommend adding the command directly to the menu, rather than through the `codeRunner`.